ingestion/src/metadata/ingestion/source/database/postgres/queries.py

```diff
@@ -72,3 +72,7 @@
      where par.relname='{table_name}' and  par.relnamespace::regnamespace::text='{schema_name}'
     """
 )
+
+POSTGRES_GET_ALL_TABLE_PG_POLICY = """
+SELECT oid, polname FROM pg_policy;
```

Comment:
this query will not be enough to generate fqn below as it does not fetch the db, schema and table_name

Reply:
Done